net-dns/dnscap/dnscap-1.10.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="network capture utility designed specifically for DNS traffic"
+HOMEPAGE="https://dnscap.dns-oarc.net/"
+
+SRC_URI="https://www.dns-oarc.net/files/dnscap/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="ISC"
+IUSE=""
+
+RDEPEND="net-libs/libpcap
+	net-libs/ldns
+	sys-libs/zlib
+	dev-libs/openssl:*
```

Comment:
This is almost certainly wrong, and usually indicates that whoever did this tried to silence repoman without trying to understand what it's saying. https://devmanual.gentoo.org/general-concepts/slotting/index.html

Reply:
fixed